services/seller.js

```diff
@@ -8,46 +8,28 @@ const models = require('../models');
 module.exports = {
 
   createSeller: (seller) => {
-    return new Promise((resolve) => {
-      bcrypt.hash(seller.password, config.bcryptSaltRounds).then((hash) => {
-        seller.password = hash;
-
-        // insert seller info to db
-        models.seller.create(seller).then(seller => {
-          resolve(seller.dataValues);
-        });
-      });
-    });
+    const hash = bcrypt.hashSync(seller.password, config.bcryptSaltRounds);
```

Comment:
We can switch to async version if we see any problems. Since we don't have tons of users so I think sync would work just fine.

Reply:
see my comment below :)